main.cpp

```diff
@@ -52,85 +52,88 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun:  Game
+//  action 1: start the game
+game.start();
+//  action 2: choose the player
+game.chooseThePlayer();
 
-//  action 2:
-
-//  action 3:
+//  action 3: run the map
+game.run();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: baby
+//  action 1: baby cry
+baby.cry();
+//  action 2: baby pop
+baby.pop();
+//  action 3: baby gets up
+baby.getsUp();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: wizard
+//  action 1: wizard fly
+wizard.fly();
+//  action 2: wizard use a potion
+wizard.usePotion();
+//  action 3: wizard do magic
+wizard.doMagic();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bomb
+//  action 1: start bomb timer
+bomb.timerStart();
+//  action 2: the timer of the bomb sounds
+bomb.soundTimer();
+
+//  action 3: disconnected the bomb
+bomb.disconnected();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bee
+//  action 1: bee fly
+bee.fly();
+//  action 2: bee take polen
+bee.takePolen();
+//  action 3: bee fight
+bee.fight();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: door
+//  action 1: the door open
+door.open();
+//  action 2: the door close
+door.close();
+//  action 3: the door lock
+door.lock
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: computer
+//  action 1: computer run the ISO
+computer.runIso();
+//  action 2: computer ask password
+computer.password();
+//  action 3: computer off
+computer.off();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.